NineChronicles.Headless/GraphQLService.cs

```diff
@@ -161,12 +169,19 @@ public void ConfigureServices(IServiceCollection services)
                     .AddLibplanetExplorer()
                     .AddUserContextBuilder<UserContextBuilder>()
                     .AddGraphQLAuthorization(
-                        options => options.AddPolicy(
-                            LocalPolicyKey,
-                            p =>
-                                p.RequireClaim(
-                                    "role",
-                                    "Admin")));
+                        options =>
+                        {
+                            options.AddPolicy(
+                                LocalPolicyKey,
+                                p =>
+                                    p.RequireClaim(
+                                        "role",
+                                        "Admin"));
+                            options.AddPolicy(
+                                JwtPolicyKey,
+                                p =>
+                                    p.RequireClaim("iss", "planetariumhq.com"));
```

Comment:
Do we fine hard coding?

Reply:
hmm, it would be better to fetch from config